AEIP-9.md

```diff
@@ -0,0 +1,76 @@
+---
+AEIP: 9
+Title: Standard NFT Metadata
+Author: Julien Leclerc <julien.leclerc05@protonmail.com>
+Type: Standard Track
+Category: AERC
+Require: AEIP-2, AEIP-8
+Status: Draft
+Created: 2023-03-12
+---
+
+## Abstract
+This AEIP defines a standardized JSON structure for the metadata of a non-fungible tokens (NFTs) on the Archethic blockchain. This AEIP builds on top of [AEIP-2](https://github.com/archethic-foundation/aeip/blob/main/AEIP-2.md), which defines the basic fields required for creating NFTs on Archethic. The new standardized JSON structure includes additional fields for specifying the content of the NFT, including its name, description, and various types of content references.
+
+## Motivation
+Currently, there is no standardized way of specifying the content of an NFT on Archethic. This creates inconsistency and potential confusion for users, and makes it difficult for developers to build NFT-related tools and applications. By defining a standardized JSON structure for NFT metadata that includes fields for specifying content, we can create a more consistent and user-friendly NFT ecosystem on Archethic.
+
+## Specification
+The JSON structure for creating an NFT on the Archethic must adhere to the guidelines set forth by [AEIP-2](https://github.com/archethic-foundation/aeip/blob/main/AEIP-2.md). It also requires to use the [AEIP-8](https://github.com/archethic-foundation/aeip/blob/main/AEIP-8.md) standard detection.
+
+In the case of a collection, the properties specified within the `properties` object are common to all NFTs in the collection.
+
+This AEIP introduces the following new fields, which can be included either within the properties object for single NFTs or as common values for a collection, or within each object of a collection list for each individual NFT:
+
+- `type_mime` (string): The MIME type of the NFT content, defined according to [RFC 2046](https://www.rfc-editor.org/rfc/rfc2046) [(IANA list)](https://www.iana.org/assignments/media-types/media-types.xhtml).
+- `content` (object): The content of the NFT, which must contain at least one of the following fields:
+  - `raw` (string): The base64 encoding of the NFT content.
```

Comment:
it could be interesting to specify if raw is compressed or not and with which algo

Reply:
See with @Neylix . This AEIP is not concerned by compression. Ok for me.